pkg/apis/autoscaling/v2beta1/conversion.go

```diff
@@ -212,7 +212,7 @@ func Convert_v2beta1_PodsMetricSource_To_autoscaling_PodsMetricSource(in *autosc
 }
 
 func Convert_autoscaling_ExternalMetricStatus_To_v2beta1_ExternalMetricStatus(in *autoscaling.ExternalMetricStatus, out *autoscalingv2beta1.ExternalMetricStatus, s conversion.Scope) error {
-	if &in.Current.AverageValue != nil {
+	if in.Current.AverageValue != nil {
```

Comment:
To be sure I'm understanding, are you saying its always going to be true, because & references the pointer, and the pointer will never be nil?

At a higher level, do we even need this `if` statement? I assume the uninitialized value of `out.CurrentAverageValue` is nil, so if we don't assign to it, it will remain `nil`... so can we just remove the `if` statement entirely?

Reply:
The if statement has the effect of not overwriting a non-nil out.CurrentAverageValue with nil value.

Reply:
Why would it already be set (non-nil) in a conversion context?  Conversion is supposed to produce an "equivalent" not a "merge"